### PR TITLE
Move gui_qt src to client

### PR DESCRIPTION
**Pull request by NIKEA-SOFT**
All files from gui_qt have been moved to the parent client directory
#915

### CODE REVIEW

**Review by lmoureaux — COMMENTED** [review]
Looks good! I only have a very small comment.

We'll want to drop references to gui-qt when including headers and also drop libclient, but let's keep this for other PRs.

File: docs/Coding/hacking.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ The source code has the following important directories:
 * :file:`common`: data structures and code used by both the client and server.
 * :file:`server`: (duh)
 * :file:`client`: common client code
-* :file:`client/gui-qt` Qt gui implementation of the client.
+* :file:`client`: Qt gui implementation of the client.
```
Comment (lmoureaux) [inline docs/Coding/hacking.rst:20]: Merge it with the line above

**Comment by lmoureaux** [comment]
Could you move the server fix to another PR? I have some comments about it and it's totally unrelated to `gui_qt`.

File: docs/Coding/hacking.rst (hunk 1 above)
Comment (NIKEA-SOFT) [reply docs/Coding/hacking.rst]: > Merge it with the line above

Done

**Comment by NIKEA-SOFT** [comment]
> Could you move the server fix to another PR? I have some comments about it and it's totally unrelated to `gui_qt`.

Ok

File: translations/core/POTFILES.in [hunk 2]
```diff
@@ -16,48 +16,48 @@ client/connectdlg_common.cpp
...
+client/shortcuts.cpp
+client/spaceshipdlg.cpp
+client/themes.cpp
+client/tooltips.cpp
+client/top_bar.cpp
+client/tradecalculation.cpp
+client/unitreport.cpp
+client/unitselect.cpp
+client/voteinfo_bar.cpp
+client/widgetdecorations.cpp
```
Comment (lmoureaux) [inline translations/core/POTFILES.in:60]: Hadn't seen it last night, can you sort it alphabetically?

Comment (NIKEA-SOFT) [reply translations/core/POTFILES.in:60]: > Hadn't seen it last night, can you sort it alphabetically?

Of course

**Comment by lmoureaux** [comment]
Thanks!

**Pull request merged**